lib/app/features/users/data/models/user_detail_profile_model.dart

```diff
@@ -8,23 +9,65 @@ class UserDetailProfileModel extends UserDetailProfileEntity {
     String? miniBio,
     String? skills,
     String? activity,
+    List<UserDetailBadgeEntity>? badges,
   }) : super(
           nickname: nickname,
           avatar: avatar,
           clientId: clientId,
           miniBio: miniBio,
           skills: skills,
           activity: activity,
+          badges: badges,
         );
 
   factory UserDetailProfileModel.fromJson(Map<String, dynamic> jsonData) {
+    List<UserDetailBadgeEntity>? badges;
```

Comment:
Eu tenho uma tendência de tratar coletivos com elementos vazio se não tiver dados (sei q na migração foi ativando um monte de coisa como nullable, mas não é o foco aqui). O que significa isto? Eu prefiro ter 

```suggestion
   var badges = <UserDetailBadgeEntity>[];
```

Por que eu prefiro isto? Porque facilita a validação do código no futuro. Assim é mais fácil saber se devo entrar numa fluxo ou não.

```dart
  if (badges.isEmpty) {
     // faço algo
  }
```

Com o nullable, tenho que realizar duas operações, e deixa a leitura mais complexa

```dart
  if (badges == null && badges.isEmpty) {
     // faço algo
  }
```

Considere esta opção de não utilizar nullable para collections, o `isEmpty` or `isNotEmpty` está aí para ajudar a facilitar a leitura do código

Reply:
Fiquei na duvida se seguia o padrao dos outros atributos nullable e acabei seguindo, mas tambem prefiro tratar com o isEmpty ou isNotEmpty. Vou fazer essas alteracoes.